text/0000-clarity-api-description.md

```diff
@@ -41,10 +41,7 @@ This type of transition, though, should have a URL and a method.
 
 We should add a section that defines what media types the server will respond with.
 
-# Unresolved questions
-
 ## Transition Attributes
 
-I think it would be helpful to call this something other than attributes, as this will not be obvious to most people.
-We have parameters, variables, and attributes throughout the document, which seems confusing.
-What should this be called?
+We should rename transition attributes to "data" to prevent confusion because the term "attributes" is already used within Refract.
```

Comment:
That's already address in the separate point (https://github.com/refractproject/rfcs/pull/1/files#diff-755b54829c9d98b897b8c1da6fb0b33aR20). Where we're going to rename them as `hrefVariables`.